waspc/tools/ensure_todoapp_builds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+# Compiles the todoApp and checks `npm run build` on both client and server.
```

Comment:
Maybe provide a bit of context of why we care about this. You can explain that this is what is done during deploymen and that we want to make sure we have a way to test it without doign actual deployment, or smth like that?